spec/features/users_can_export_spending_breakdown_spec.rb

```diff
@@ -16,6 +18,23 @@
       expect(page).to have_content(export_in_progress_msg)
     end
 
+    context "when a fund has an uploaded spending breakdown" do
+      let(:newton_fund) { Fund.by_short_name("NF").activity }
+
+      before do
+        newton_fund.update!(spending_breakdown_filename: "spending_breakdown-20230130120000.csv")
+      end
```

Comment:
Do we need to create all the funds and then update Newton or could we just create Newton in this context?

Reply:
The view iterates through all the `Fund`s and inspects their fund activities for the existence of a `spending_breakdown_filename`. A nil guard wouldn't help, because the `Fund` raises `ActiveError::RecordNotFound` if the activity doesn't exist.